src/workerd/api/streams/standard.c++

```diff
@@ -3019,7 +3044,7 @@ private:
           }), [](auto& js, jsg::Value exception) mutable -> Result {
             return kj::mv(exception);
           }).then(js, ioContext.addFunctor(
-              [readable=kj::mv(readable),pumpToReader=kj::mv(pumpToReader)]
+              [readable=kj::mv(readable),pumpToReader=kj::mv(pumpToReader),&handleException]
```

Comment:
Isn't this taking a reference to a stack variable that will be UAF by the time the lambda executes?